src/main/java/seedu/pill/command/HelpCommand.java

```diff
@@ -41,44 +49,125 @@ private void showGeneralHelp() {
         System.out.println("  help    - Shows this help message");
         System.out.println("  add     - Adds a new item to the list");
         System.out.println("  delete  - Deletes an item from the list");
+        System.out.println("  edit    - Edits quantity of an existing item in the list");
         System.out.println("  list    - Lists all items");
         System.out.println("  quit    - Exits the program");
         System.out.println("Type 'help <command>' for more information on a specific command.");
+        System.out.println("Add  '-v' after the command for verbose output with examples.");
     }
 
     /**
-     * Shows detailed help information for each command.
-     * @param command - optional user input that determines which information is displayed.
+     * Calls the appropriate method depending on what the user requests help for.
+     * @param command - optional user input that determines which help information is displayed to the user.
      */
     private void showSpecificHelp(String command) {
         switch (command.toLowerCase()) {
         case "help":
-            System.out.println("help: Shows help information about available commands.");
-            System.out.println("Usage: help [command]");
-            System.out.println("  [command] - Optional. Specify a command to get detailed help.");
+            showHelpHelp();
             break;
         case "add":
-            System.out.println("add: Adds a new item to the inventory.");
-            System.out.println("Usage: add <name> <quantity> <price>");
-            System.out.println("  <name>     - Name of the item");
-            System.out.println("  <quantity> - Initial quantity of the item");
+            showAddHelp();
             break;
         case "delete":
-            System.out.println("delete: Removes an item from the inventory.");
-            System.out.println("Usage: delete <index>");
-            System.out.println("  <name> - Name of the item to delete (as shown in the list)");
+            showDeleteHelp();
+            break;
+        case "edit":
+            showEditHelp();
             break;
         case "list":
-            System.out.println("list: Displays all items in the inventory.");
-            System.out.println("Usage: list");
+            showListHelp();
             break;
         case "quit":
-            System.out.println("quit: Exits the program.");
-            System.out.println("Usage: quit");
+            showQuitHelp();
             break;
         default:
             System.out.println("Unknown command: " + command);
-            System.out.println("Type 'help' for a list of available commands.");
```

Comment:
I appreciate the decision to abstract out the print implementation, makes out code more readable.